### PR TITLE
Update Prebuilt Rule Links for Malicious Site in 7.10

### PR DESCRIPTION
**Pull request by terrancedejesus**
## Summary
Fixes links to malicious site.

Related:
* https://github.com/elastic/security-docs/pull/4239
* https://github.com/elastic/detection-rules/pull/3271

### CODE REVIEW

File: docs/detections/prebuilt-rules/rule-details/remote-file-copy-via-teamviewer.asciidoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ Identifies an executable or script file remotely downloaded via a TeamViewer tra
 
 *References*:
 
-* https://blog.menasec.net/2019/11/hunting-for-suspicious-use-of.html
+* http://web.archive.org/web/20230329160957/https://blog.menasec.net/2019/11/hunting-for-suspicious-use-of.html
```
Comment (stiltz) [inline docs/detections/prebuilt-rules/rule-details/remote-file-copy-via-teamviewer.asciidoc:25]:
```suggestion
* https://web.archive.org/web/20230329160957/https://blog.menasec.net/2019/11/hunting-for-suspicious-use-of.html
```

File: docs/detections/prebuilt-rules/rule-details/suspicious-managed-code-hosting-process.asciidoc [hunk 2]
```diff
@@ -22,7 +22,7 @@ Identifies a suspicious managed code hosting process which could indicate code i
 
 *References*:
 
-* https://blog.menasec.net/2019/07/interesting-difr-traces-of-net-clr.html
+* http://web.archive.org/web/20230329154538/https://blog.menasec.net/2019/07/interesting-difr-traces-of-net-clr.html
```
Comment (stiltz) [inline docs/detections/prebuilt-rules/rule-details/suspicious-managed-code-hosting-process.asciidoc:25]:
```suggestion
* https://web.archive.org/web/20230329154538/https://blog.menasec.net/2019/07/interesting-difr-traces-of-net-clr.html
```

**Review by stiltz — CHANGES_REQUESTED** [review]
updated `http://` to `https://` I verified the archive.org site loads over https.

**Comment by terrancedejesus** [comment]
@stiltz - Thanks Ill update the rest of the PRs with this!

**Review by stiltz — APPROVED** [review]
lgtm

**Pull request merged**